.github/workflows/main.yml

```
@@ -20,11 +20,14 @@ jobs:
           - Django>=4.2,<5.0
           - Django>=5.0,<5.1
           - Django>=5.1,<5.2
+          - Django>=5.2rc1,<6.0
         exclude:
           - python-version: "3.9"
             django-version: "Django>=5.0,<5.1"
           - python-version: "3.9"
             django-version: "Django>=5.1,<5.2"
+          - python-version: "3.9"
+            django-version: "Django>=5.2rc1,<6.0"
```

Update this once the final release is out

```suggestion
            django-version: "Django>=5.2,<6.0"
```

---

Feel free to add that and to remove the draft status from this PR and I'll move on to merge it. :slightly_smiling_face:

---

> Feel free to add that and to remove the draft status from this PR and I'll move on to merge it. 🙂

Alright, I think it's all ready now